modules/custom/social_ajax_comments/social_ajax_comments.module

```diff
@@ -137,21 +137,19 @@ function social_ajax_comments_comment_links_alter(array &$links, CommentInterfac
   // Url::fromRoute()->toRenderable doesn't fully take care of access checks.
   // We need the custom checks to override what Ajax comments does for us.
   // Mainly because of UX flaws in the edit form that opens up.
-  if ($entity->isPublished() && $account->hasPermission('administer comments')) {
```

Comment:
Since LU doesn't have administer comments permission, this was available for LU. Moved the comment-edit link alter outside of that scope and that should do the trick, if the entity isn't published should also be no problem :) only those who can edit the entity can see the links so that is still valid.

Reply:
That seems fine to me.